### ISSUE TITLE
Add license name if missing from license text

### ISSUE THREAD

**Issue by jayvdb** [TobyAndToby/generate-license-file#526]
**How are you using generate-license-file?**
Library

**Is your feature request related to a problem? Please describe.**
Many license text files do not explicitly mention the license name/spdx identifier because it is unnecessary in the context of the source repo where the spdx identifier is in the package.json, and Github now shows the license spdx.

However when the license text is used in a different context, it is helpful for the license name/spdx to be included.
Then the reader does not need to be able to manually convert the license text into the license name, which can be rather tricky for some of licenses that are quite close.

**Describe the solution you'd like**
This tool could check whether the license name/identifer is not included in the license text, and add it.

**Additional context**
e.g. https://github.com/Raynos/function-bind/blob/master/LICENSE

**Comment by tobysmith568** [comment]
Hey @jayvdb,

It's important to us that we don't modify the licenses of any packages as we believe the licenses should be handled verbatim.

For us to provide the license name or SPDX identifier for a license (either in the content itself or as additional data) we would need to be guessing based on what we _think_ the license file contains. While some websites and tools provide this functionality (even GitHub do it), it something we're not willing to do as we'd prefer to rely on what the package authors have explicitly declared.

**Comment by jayvdb** [comment]
@tobysmith568 ,

> It's important to us that we don't modify the licenses of any packages as we believe the licenses should be handled verbatim.

Ya, I agree.

When I recreated this issue, there was only a "content" field, and was didnt dare ask for a new field, so I suggested enhancing the "content" field.

But in light of https://github.com/TobyAndToby/generate-license-file/issues/529 , I think a better solution would be to add an "spdx" field.

Then I can do my presentation enhancements in my presentation layer.
